js/src/main/scala/scala/json/ast/JValue.scala

```diff
@@ -31,6 +31,34 @@ sealed abstract class JValue extends Product with Serializable {
   def toJsAny: js.Any
 }
 
+@JSExport
+object JValue {
+  /**
+    * Converts a Javascript object/value coming from Javascript to a [[JValue]].
```

Comment:
Can you say Scala.js Javascript object/value rather than just Javascript object/value. This is to clear confusion from Scala.js Javascript versus Javascript alone (which the AST also supports)